plugins/jobs/src/js/JobsOverviewRouteProvider.tsx

```diff
@@ -0,0 +1,46 @@
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import { routerShape, LocationDescriptor } from "react-router";
+import JobsOverview from "#PLUGINS/jobs/src/js/JobsOverview";
+
+interface JobsOverviewRouteProviderProps {
+  params: { id: string };
+  location: LocationDescriptor;
+  children: React.ReactElement<any>;
+}
+
+export default class JobsOverviewRouteProvider extends React.Component<
```

Comment:
In general (this is not blocking) I would like to see this route provider more as a general thing. It could expose the router and the location to the child components. This way it would be reusable and not entangled with the JobsOverviewRouteProvider. But we can do this as a separate refactoring.

Reply:
agree, but we might want to double check if this is something we _want_ to have a a general thing.